packages/api/lib/writeRecords/write-granules.js

```diff
@@ -204,7 +204,7 @@ const _writeGranuleFiles = async ({
 }) => {
   let fileRecords = [];
 
-  if (status !== 'running') {
+  if (status !== 'running' && status !== 'queued') {
```

Comment:
Does this generate a file record for the granule if it's in failed status?

Reply:
🤔 That's a good question. It would seem so